### ISSUE TITLE
older versions. . .

### ISSUE THREAD

**Issue by NightRidr** [GeekGene/Community-Weaver#1]
As the newest version of CW (3) appears to not be open-source (the timebank refused to send the source code?) I must revert to this older version of CW.  I am not well versed in Drupal and have just enlisted a Drupal Service company.  My concern is they will not be able to use such an older version of modules and such without a good deal of reverse engineering?

Has anyone successfully installed this and be willing to assist me (remote login) with my server box?  (Of course I will pay)
Don

**Comment by artbrock** [comment]
Updating the modules to more recent versions should still be workable. And Drupal 6.x has still been receiving security updates. It would be good for you to have somebody with some drupal experience get you going though. TimebanksUSA was never too big on making the code open source so documentation beyond the included installation guide is a bit sparse.  It looks like they still have member and coordinator manuals available on http://actionhub.timebanks.org/ for using the system once you get it built and updated.

**Comment by artbrock** [comment]
I'm not available to spend time on building something on your server box, but might be able to answer some questions if your Drupal service guys get stuck on something.

**Comment by NightRidr** [comment]
artbrock, my drupal guy got me up and running; anyway i can see my server on the web?  Now comes some tweaking and I could use some help?  The CW drupal module left these menu tabs:
HOME and GIVE & RECEIVE and MY ACCOUNT (these 3 seem to be for all users) and CONTROL PANEL!

There appears to be some link activity on the first three but my main concern is I can't get the CONTROL PANEL links to show up for my coordinators?  As admin, I have an "EDIT PANEL" sub-menu which is sort of greyed out?  There are sub links which seem active?  How do I make the CONTROL PANEL active for my Coordinators and such?  As it is, my Coordinators can't do anything such as work with users, configure the web pages and tweak them for our particular message and issues. . .

**Comment by NightRidr** [comment]
my 'drupal guy' got my TB site up.  But we can't get any of the pages to work?  We can't edit pages or work with any new users or such?  Is there a way to 'seemingly activate' the site?  We can see the links, but they appear greyed out  or non-functioning?. .

**Comment by NightRidr** [comment]
We got it working.

**Comment by artbrock** [comment]
Hooray!

**Comment by GeoMaciolek** [comment]
> And Drupal 6.x has still been receiving security updates.

For what it's worth for people viewing this issue now - as of February 2016, [Drupal 6 is end-of-life](https://www.drupal.org/drupal-6-eol) and no longer receiving updates.

I'm quite perplexed about this whole thing; it seems super strange to me that Community Weaver 3 is - as far as I can tell - not open source.

They are technically allowed to do this; it's the [web services loophole](http://buytaert.net/long-live-the-web-services-loophole).  But it does seem to be counter to their various stated goals.

In fact, even in the [Community Weaver 3 Announcement](http://timebanks.org/community-weaver-3-has-arrived/):

> Reciprocity is our survival and our path to sustainability, and it is based on trust. In the spirit of trust and the Gift, Community Weaver is our gift to Timebankers everywhere, and we offer it in the Gift Economy.  You may give your time, your money, and your infectious enthusiasm in Reciprocity.

The whole thing is super confusing to me. One would think that the sorts of folks who want to encourage the kinds of community & economy that Time Banks use, and who make use of Open Source Software would want to contribute that back... especially given they say so!

**Comment by artbrock** [comment]
I agree. That's part of why my arrangement with them was that we could release it open source. 

I think their challenge is that they've tied their business model to charging timebanks for hosting the software and if they release the software for free, then they lose their business model. The thing is that they're not a software company, so I thought they'd fare better building an open source community around the software, but so far they haven't been able to risk their business model.  I keep hoping they'll get there... and if they don't, they may just get left behind when others do build an open source community.
